### ISSUE TITLE
Gene annotation file not rendered correctly

### ISSUE THREAD

**Issue by alexpreynolds** [higlass/higlass#295]
Local installation at: http://hg-dev.altius.org/app/

_Commit_: 

Tag v0.6.31 of higlass-website
Master branch of higlass-server
Tag v0.9.1 of clodius

#### Steps to reproduce
1. Annotations are converted and ingested (cf. issue #294 for details; using higlass-docs as basis for conversion/ingestion)

#### Observed behavior
- Annotation ID name is not shown
- Annotations are shown with reverse-strand directionality
- Overlapping annotations overwrite each other

As an example, please see the "my_annotations.db" track at the bottom-most row of this image:

![render](https://user-images.githubusercontent.com/33584/43600893-bf62c43a-9640-11e8-9dd9-df4df6392446.png)

#### Expected behavior
- We would expect to see annotation ID names
- Annotations where the strand field is marked `.` (neither forward `+` or reverse `-`) may be better to be rendered without directionality color or arrow cues
- It would be helpful to show overlapping annotations in multiple rows, similar to full or pack display mode in the UCSC Genome Browser.

**Comment by pkerpedjiev** [comment]
So for now we don't support multiple rows of gene annotations. If you're willing to share your beddb file I can take a look and try to diagnose the issue.

**Comment by alexpreynolds** [comment]
Absolutely! Please use the following, or similar, to download the beddb file:

```
$ wget -O- --user encode --password collabor8 https://resources.altius.org/~areynolds/proj/wouter/072418/data/masterlist_DHSs_733samples_WM20180608_all_colorsMax_higlass_resort_sig_maxpertile100.db > /tmp/masterlist_DHSs_733samples_WM20180608_all_colorsMax_higlass_resort_sig_maxpertile100.db
```

Please let me know if I can provide any other information, and thanks!

**Comment by snystrom** [comment]
Multiple rows would also be very helpful for overlapping BED regions. An option similar to IGV's 'expanded' view would be useful for producing staggered display of overlapping regions along the short axis. I'm also running into this issue but with bed-like files.

**Comment by pkerpedjiev** [comment]
@snystrom We should actually have this functionality.  Would you mind sharing your beddb file?

@alexpreynolds I completely forgot about this issue. Do you know if it's still a problem?

**Comment by alexpreynolds** [comment]
I have not tested this issue with the v1.4.0 higlass plugin. I will try to retest this issue and see what has changed or remained the same.

I think, separately, there may still be an issue with scalability (https://github.com/higlass/higlass/issues/294) when requesting large numbers of annotations from our higlass server instance, but I will try this out again and update that ticket, as well.

**Comment by snystrom** [comment]
@pkerpedjiev [Here's my beddb file](https://www.dropbox.com/s/gg5ov3n26kwgg9h/dm3_bands_liftover.bed.beddb?dl=0). The issue I have is that these regions are 1bp apart so they would look better if they could be rendered in a more staggered fashion like this:

![image](https://user-images.githubusercontent.com/17264973/52804594-982c4380-3052-11e9-82cc-42d0666727c3.png)
